### DISCUSSION TITLE
Add Yaml language support

### DISCUSSION THREAD

**Discussion by Th3S4mur41** [biomejs/biome#199] [Ideas]
Following up on the [same request in rome](https://github.com/rome/tools/discussions/4336)

With all the languages above supported, there would be one that is still missing to entirely replace prettier in most projects: YAML
Would you consider adding YAML to the list of languages that Biome can format?

![image](https://github.com/biomejs/biome/assets/5543595/4515a72d-68c1-4686-aaf7-ee6ba54089d5)

**Comment by ematipico** [comment]
It's definitely outside our roadmap for now, or, to phrase it differently, it has the lowest priority among the web languages.

 Once all the previous languages are supported, we could start working on YAML. Still, we could add it

**Reply by scamden** [reply]
just to add signal this means we have to keep prettier in our stack and bifurcate all our formatting paths to send yaml files to it. (same for CSS and MD which looks like are on your radar already)

**Reply by scamden** [reply]
also holy wow is biome faster. even with using prettier for yaml files our full repo format is like 30-50x faster depending how you count it. so thank you :)
